book-2-levelup/chapters/DRF_INSTALLS.md

```diff
@@ -77,6 +77,18 @@ If that command throws errors, try this one.
 python3 manage.py startapp levelupapi
 ```
 
+## Add Content To .gitignore File
+
+Create a `.gitignore` file by running the following command
+
+```sh
+touch .gitignore
```

Comment:
Let's add the curl command to populate the .gitignore for them so they don't have to search and copy/paste
```suggestion
touch .gitignore
curl -L -s 'https://raw.githubusercontent.com/github/gitignore/master/Python.gitignore' > .gitignore
```